scripts/babel/transform-object-assign-require.js

```diff
@@ -28,6 +28,10 @@ module.exports = function autoImporter(babel) {
 
     visitor: {
       CallExpression: function(path, file) {
+        if (file.filename.indexOf('object-assign') !== -1) {
+          // Don't replace Object.assign if we're transforming object-assign
+          return;
```

Comment:
Since we're now transforming the file itself, I needed to ensure this bails out.